_component_design/chips.md

```diff
@@ -13,32 +13,87 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
+## Introduction
+
 <div class="hxRow">
-{% column left:"hxCol" %}
-{% figure [caption:"Basic"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/chips.svg){:width="100%"}
-{% endfigure %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- Chips are best used with indicating a list of items that may be part of a different workflow. For example, requestors on tickets can be added and removed or management tasks that can be performed by requestors. Devices also offer a good option for chip use. 
+
+
+### Best practices
+
+- Avoid using chips in multiply locations on apage. 
+- Make sure the user is well aware of what the chip is. 
+- Using the expanded version can help ensure the information needed by the user is visible.
+
 {% endcolumn %}
-{% column left:"hxCol" %}
-{% figure [caption:"With Icon"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/chips-icon.svg){:width="100%"}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"The chip component"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/chips/chips-hero.png){: width="1440px"}
 {% endfigure %}
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Delete"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/chip-delete.svg){:width="100%"}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Default chip
+
+Default chip is a single circle that contains a preview of the item the chip represents. This can be either letters to indicate the name, an image to indicate a user, or a product icon to indicate the type of device.
```

Comment:
Define `This` more clearly.

Reply:
Does this read better, "The preview can consist of the the following; letters to indicate the name, an image to indicate a user, or a product icon to indicate the type of device."